knowledge-base/scheduler-kb-context-menu-on-appointments-and-empty-slots.md

```diff
@@ -0,0 +1,367 @@
+---
+title: Manage Appointments via Context Menu in the Scheduler
+description: How to use the context menu to add, edit, delete appointments in specific timeslots in the Scheduler.
+type: how-to
+page_title: How to Add Appointments on Empty Slot in the Scheduler Using the ContextMenu. How to Add ContextMenu on Appointment in the Scheduler.
+slug: scheduler-kb-context-menu-on-appointments-and-empty-slots
+tags: scheduler, blazor, contextmenu, appointments, empty slots
+res_type: kb
+ticketid: 1656007
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>
+                ContextMenu for Blazor,<br />
+                Scheduler for Blazor
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+This KB article answers the following questions:
+
+* How to add a [Context menu]({%slug contextmenu-overview%}) to the appointments to provide shortcuts to custom features?
+* How to add a Context menu that can be opened anywhere on the scheduler board, not just on appointments? 
+* Is it possible to get the timeslot information where the Context menu is opened?
```

Comment:
Does this sentence make sense in the Blazor context, because outside of it, it's hard to understand? It could be a matter of a missing preposition or an erroneous word order. 

Would this make sense instead? "Is it possible to get the timeslot for which the Context menu is opened?"